app/dash/Tokens/CustomTokens/index.js

```diff
@@ -25,6 +26,11 @@ class CustomTokens extends React.Component {
                   return a.chainId <= b.chainId
                 })
                 .map((token, i) => {
+                  const { address, chainId, media } = token
+                  const isNative = address === NATIVE_CURRENCY
+                  const chainColor = this.store('main.networksMeta.ethereum', chainId, 'primaryColor')
+                  const isEth = isNative && [1, 3, 4, 5, 10, 42, 42161, 11155111].includes(chainId)
```

Comment:
do we have this defined somewhere as a utility function? is it worth extracting out to use elsewhere?

Reply:
yeah, probably worth extracting at this point